packages/shoreline/src/components/search/tests/search.test.tsx

```diff
@@ -30,6 +31,16 @@ describe('search', () => {
     expect(container.querySelector('[data-sl-spinner]')).toBeInTheDocument()
   })
 
+  it('should show have default placeholder localized', () => {
```

Comment:
I've just noticed that the tests aren't using the new pattern. The visual tests should be made using Storybook instead of unit tests, you can check our documentation for guidance.
* [Storybook guidelines](https://shoreline.vtex.com/guides/code/storybook-guideline#tests)
* [Tests guidelines](https://shoreline.vtex.com/guides/code/test-guideline)

Reply:
I had seen this but I didn't think location was included as a visual. I'll change it